frontend/app/routes/$lang+/_protected+/personal-information+/index.tsx

```diff
@@ -174,6 +181,8 @@ export default function PersonalInformationIndex() {
             </p>
           )}
         </DescriptionListItem>
+        <DescriptionListItem term={t('personal-information:index.marital-status')}>{`${maritalStatus ? maritalStatus : t('personal-information:index.no-marital-status')} `}</DescriptionListItem>
+        <DescriptionListItem term={t('personal-information:index.date-of-birth')}> {`${birthParsedFormat} `}</DescriptionListItem>
```

Comment:
**Nitpick:** is there a reason for the spacing here? Shouldn't this be:

``` tsx
<DescriptionListItem term={t('personal-information:index.date-of-birth')}>{birthParsedFormat}</DescriptionListItem>
```

Reply:
Yeah, that was accidental added. Pushed: https://github.com/DTS-STN/canadian-dental-care-plan/commit/c643167653705749ea9866676ea2487dc7f57182